### PR TITLE
Update NFC article with spec changes

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR about the Web NFC article with latest spec and implementation changes:

- Replace `NDEFWriter.write` with `NDEFReader.write`
- Removes NDEF Scan options for filtering
- Removes NDEF Write option for ignoring read
- Replaces `onerror` with `onreadingerror`
- Marks chrome origin trial complete

Live preview: 
- https://deploy-preview-4259--web-dev-staging.netlify.app/nfc
- https://deploy-preview-4259--web-dev-staging.netlify.app/fugu-status

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
@jpmedley Can you review?

File: src/site/content/en/blog/nfc/index.md [hunk 1]
```diff
@@ -74,45 +74,39 @@ Examples of sites that may use Web NFC include:
...
 ### Feature detection {: #feature-detection }
 
 Feature detection for hardware is different from what you're probably used to.
-The presence of `NDEFReader` and `NDEFWriter` tells you that the browser
-supports Web NFC, but not whether the required hardware is present. In
-particular, if the hardware is missing, the promised returned by certain calls
-will reject. I'll provide details when I describe `NDEFReader` and `NDEFWriter`.
+The presence of `NDEFReader` tells you that the browser supports Web NFC, but
+not whether the required hardware is present. In particular, if the hardware is
+missing, the promised returned by certain calls will reject. I'll provide
```
Comment (mihajlija) [inline src/site/content/en/blog/nfc/index.md:105]:
```suggestion
missing, the promise returned by certain calls will reject. I'll provide
```

**Review by mihajlija — APPROVED** [review]

File: src/site/content/en/blog/nfc/index.md [hunk 2]
```diff
@@ -74,45 +74,39 @@ Examples of sites that may use Web NFC include:
 | 1. Create explainer                        | [Complete][explainer]        |
 | 2. Create initial draft of specification   | [Complete][spec]             |
 | **3. Gather feedback and iterate design**  | [**In Progress**](#feedback) |
-| **4. Origin trial**                        | [**In Progress**][ot]        |
+| 4. Origin trial                            | Complete                     |
```
Comment (jpmedley) [inline src/site/content/en/blog/nfc/index.md:77]: Is NFC shipping in 88?

File: src/site/content/en/blog/nfc/index.md [hunk 3]
```diff
@@ -74,45 +74,39 @@ Examples of sites that may use Web NFC include:
...
 | 5. Launch                                  | Not Started                  |
 
 </div>
 
+{% Aside %}
+  During the origin trial phase, there was a `NDEFWriter` object that has been
```
Comment (jpmedley) [inline src/site/content/en/blog/nfc/index.md:83]:
```suggestion
  During the origin trial phase, there was an `NDEFWriter` object that has been
```

File: src/site/content/en/blog/nfc/index.md [hunk 4]
```diff
@@ -74,45 +74,39 @@ Examples of sites that may use Web NFC include:
...
+  merged into `NDEFReader`. There were a number of other [changes] that you can
+  read up on.
```
Comment (jpmedley) [inline src/site/content/en/blog/nfc/index.md:85]: Link?

File: src/site/content/en/blog/nfc/index.md [hunk 5]
```diff
@@ -74,45 +74,39 @@ Examples of sites that may use Web NFC include:
...
+{% endAside %}
+
 ## Using Web NFC {: #use }
 
 ### Enabling via chrome://flags
 
-To experiment with Web NFC locally on Android, without an origin trial token,
-enable the `#experimental-web-platform-features` flag in `chrome://flags`.
+To experiment with Web NFC locally on Android, enable the
```
Comment (jpmedley) [inline src/site/content/en/blog/nfc/index.md:92]:
```suggestion
To experiment with Web NFC on Android, enable the
```

File: src/site/content/en/blog/nfc/index.md [hunk 6]
```diff
@@ -162,17 +154,17 @@ granted. The promise will resolve if the following conditions are all met:
 
 Once the promise is resolved, incoming NDEF messages are available by
 subscribing to "reading" events via an event listener. You should also subscribe
-to "error" events to be notified when incompatible NFC tags are in proximity
-range.
+to "readingerror" events to be notified when incompatible NFC tags are in
```
Comment (jpmedley) [inline src/site/content/en/blog/nfc/index.md:157]:
```suggestion
to `readingerror` events to be notified when incompatible NFC tags are in
```

File: src/site/content/en/blog/nfc/index.md [hunk 7]
```diff
@@ -162,17 +154,17 @@ granted. The promise will resolve if the following conditions are all met:
...
+proximity range.
```
Comment (jpmedley) [inline src/site/content/en/blog/nfc/index.md:158]:
```suggestion
proximity.
```

File: src/site/content/en/blog/nfc/index.md [hunk 8]
```diff
@@ -441,20 +381,19 @@ if (permissionStatus.state === "granted") {
 
 Using the <code>[AbortController]</code> primitive makes it easy to abort NFC operations. The
 example below shows you how to pass the `signal` of an AbortController through
```
Comment (jpmedley) [inline src/site/content/en/blog/nfc/index.md:383]:
```suggestion
example below shows you how to pass the `signal` of an `AbortController` through
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/nfc/index.md (hunk 2 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/nfc/index.md:77]: Nope.

File: src/site/content/en/blog/nfc/index.md (hunk 3 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/nfc/index.md:83]: Done.

File: src/site/content/en/blog/nfc/index.md (hunk 4 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/nfc/index.md:85]: Thanks for catching! Fixed.

File: src/site/content/en/blog/nfc/index.md (hunk 5 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/nfc/index.md:92]: Done

File: src/site/content/en/blog/nfc/index.md (hunk 6 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/nfc/index.md:157]: Done. I've fixed `reading` as well above.

File: src/site/content/en/blog/nfc/index.md (hunk 7 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/nfc/index.md:158]: Done

File: src/site/content/en/blog/nfc/index.md (hunk 8 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/nfc/index.md:383]: Done

**Review by beaufortfrancois — COMMENTED** [review]
Your feedback has been addressed. Thank you!

**Review by jpmedley — APPROVED** [review]

**Comment by jpmedley** [comment]
Is there any reason not to merge this now?

**Comment by beaufortfrancois** [comment]
Yes. Spec changes have not been merged yet in Chromium.

**Pull request merged**